roles/boot_iso/tasks/pxe.yml

```
@@ -0,0 +1,119 @@
+---
+
+- name: prepare PXE
+  block:
+    - name: ensure the required packgaes installed
+      package:
+        name: "{{ item }}"
+        state: present
+      with_items:
+        - syslinux
+        - tftp-server
+        - dnsmasq
+        - ipmitool
+        - httpd
+        - python3-pip
+
+    - name: Install multi python packages with version specifiers
+      pip:
+        name:
+          - pyghmi
+
+    - name: "httpd  Configuration File"
```

Maybe you can name it like
```suggestion
    - name: "Create httpd configuration file"
```